distribution/xfer/download.go

```diff
@@ -263,7 +263,7 @@ func (ldm *LayerDownloadManager) makeDownloadFunc(descriptor DownloadDescriptor,
 
 			selectLoop:
 				for {
-					progress.Updatef(progressOutput, descriptor.ID(), "Retrying in %d seconds", delay)
+					progress.Updatef(progressOutput, descriptor.ID(), "Retrying in %d second%s", delay, (map[bool]string{true: "", false: "s"})[delay == 1])
```

Comment:
TIL a new trick :)
If others want to fix this, you can improve it by just doing this: `(map[bool]string{false: "s"})[delay == 1]` it won't panic: https://play.golang.org/p/wgHjLdwjSJ